setup.py

```diff
@@ -499,6 +513,8 @@ def bundle_libzmq_extension(self):
             # include defines from zeromq msvc project:
             libzmq.define_macros.append(('FD_SETSIZE', 1024))
             libzmq.define_macros.append(('DLL_EXPORT', 1))
+            libzmq.define_macros.append(('_CRT_SECURE_NO_WARNINGS', 1))
+            libzmq.define_macros.append(('ZMQ_USE_SELECT', 1))
```

Comment:
I haven't had time to test this yet, but since `ZMQ_USE_SELECT` is the only poller choice available on Windows, I guess it doesn't hurt to provide this definition.